packages/ui5-config/src/types/middlewares.ts

```diff
@@ -16,6 +16,7 @@ export interface FioriToolsProxyConfigBackend {
     path?: string;
     url: string;
     client?: string;
+    authenticationType?: string;
```

Comment:
@cianmSAP, shouldn't this be mapped to type
https://github.com/SAP/open-ux-tools/blob/96de8f25054b1a88da17398fb9a4ad2cc0043752/packages/store/src/entities/backend-system.ts#L4-L9

Reply:
I've updated this, thanks @Klaus-Keller